lib/susedistribution.pm

```diff
@@ -166,6 +166,14 @@ sub x11_start_program {
     else {
         type_string $program;
     }
+    # With match_typed we check typed text and if doesn't match - retrying
+    # Is required by firefox test on kde, as typing fails on KDE desktop runnner sometimes
+    if ($args{match_typed} && !check_screen($args{match_typed})) {
```

Comment:
This times out after 30 seconds only. I don't see a reason why we need to wait that long to detect incomplete typing. I'm not sure I like it. Definitely you should update the documentation if you add an argument.

Reply:
If we enable this option, then we want to verify the screen, so waiting 30 seconds is fine. By default second statement is not executed, as first statement is false, so don't get your point here. I'll update documentation.